### ISSUE TITLE
couldn't run clusterwenapp, cannot retrieve information from SLURM which runs on the background.

### ISSUE THREAD

**Issue by avatar-lavventura** [ysagon/clusterwebapp#1]
I have run as follows:
```
sudo NODE_ENV=development node app.js NODE_ENV=production node app.js
Express server listening on port 6000 in debug mode
```
How could I open the interface page. From: http://localhost:6000? It says: the address is restarted. 
Is there any way to update the port#?

**Comment by ysagon** [comment]
Yes, please edit this file [default.yaml](https://github.com/ysagon/clusterwebapp/blob/master/config/default.yaml)

**Comment by avatar-lavventura** [comment]
Thank you sir, I have make the change and it works on my assign port. 

The problem is on the interface I won't able to see the slurm's completed job, pending job etc (any information related the slurm), even though slurm works on the background. Is there anything I could do to link slurm and clusterwenapp between each other? @ysagon

**Comment by ysagon** [comment]
did you had a look at the log file in the logs directory to see if there is something wrong?

**Comment by avatar-lavventura** [comment]
@ysagon : 
The way I run: 
```
mkdir logs
sudo NODE_ENV=production node app.js
Express server listening on port 3016 in production mode
```

```
start iface
iface start/running, process 8279
```

on `IP:3016`: I can see the generated website but it does not feed information from the slurm running on the background.

For example when I look into job running: the output seen as: `Page opened`

```
cat /logs/express.log
Logs: 79.123.176.55 - - [Thu, 18 May 2017 07:28:03 GMT] "GET / HTTP/1.1" 200 5389 "-" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_11_1) AppleWebKit/601.2.7 (KHTML, like Gecko) Version/9.0.1 Safari/601.2.7"
```

<img width="302" alt="screen shot 2017-05-18 at 10 34 14" src="https://cloud.githubusercontent.com/assets/18537398/26191208/9e895ae4-3bb5-11e7-8e56-71e69db2018f.png">

**Comment by avatar-lavventura** [comment]
I am still stuck at this point (I am not sure where I did mistake), do you have any further guide? It cannot retrieve information from SLURM, which runs on the background. Thank you sir for your valuable time and help. 

Please note that `sinfo` returns a valid information, hence SLURM is working on the background.

```
$ sinfo
PARTITION AVAIL  TIMELIMIT  NODES  STATE NODELIST
debug*       up   infinite      4   idle ebloc[1-4]
```

The way I run:

```
 $ sudo start iface
iface start/running, process 28818
$ sudo NODE_ENV=development node app.js
```
@ysagon

**Comment by avatar-lavventura** [comment]
Any update on this issue, I would be grateful if you have a look at it again. @ysagon
